components/brave_private_new_tab_ui/common/brave_private_new_tab.mojom

```diff
@@ -12,6 +12,7 @@ interface PageHandler {
   SetDisclaimerDismissed(bool dismissed);
   GetDisclaimerDismissed() => (bool dismissed);
   GetIsTorConnected() => (bool is_connected);
+  GetIsTorDisabled() => (bool is_disabled);
```

Comment:
Looking at `TorProfileServiceFactory::IsTorDisabled(content::BrowserContext* context)`, `tor::prefs::kTorDisabled` is in local state. If its value can be changed at runtime, specifically when NTP is shown, should we have something like `NotifyTorDisabledChagned()` in `PrivateTabPage`?

Reply:
We could but I don't really think the text needs to be dynamically updated. It would appear right the next new tab page that is opened in that case.  So it's not really worth the complexity IMHO.

Reply:
Sounds fair. Thanks.